### ISSUE TITLE
Sequence steering using SAE Latents

### ISSUE THREAD

**Issue by santule** [etowahadams/interprot#44]
Hi,

I was trying out the sequence steering notebook. One thing, I am confused about is how the embeddings are used again in the ESM model. The embeddings from ESM are taken from layer 24 and then the SAE latents are manipulated. The SAE latents are then decoded to get the embeddings again.
But the new decoded embeddings are used directly in the lm_head. (in the line below)

**logits = esm_lm.lm_head(clamped + recons_error)**     

But i thought they still need to go through layer 25 - 33 before logits can be applied ? Is that not a correct understanding ? 

Regards and Thanks.
